module/friendship/app/command/connect_friendship_test.go

```diff
@@ -33,6 +33,8 @@ func TestFriendship_ConnectFriendship(t *testing.T) {
 	friendship := domain.Friendship{UserID: friends[0], FriendID: friends[1]}
 
 	errDB := errors.New("some error from db")
+	var mapNil map[string]string = nil
+	var sliceNil []string = nil
```

Comment:
do not declare variable with null value just for returning purposes ! You can return `nil` instead

Reply:
Can not compare because a `nil` different `([]string) nil`